### PR TITLE
JP-2090: Update set_velocity_aberration to use DataModel

### LINKED ISSUE #6053: Update set_velocity_aberration to use DataModel
_Issue [JP-2090](https://jira.stsci.edu/browse/JP-2090) was created on JIRA by [Jonathan Eisenhamer](https://jira.stsci.edu/secure/ViewProfile.jspa?name=eisenham):_

`set_velocity_aberration` uses `astropy.io.fits`. Update to use `jwst.datamodels`

As a subtask, explore other similar scripts and also update.

As a subtask, `set_telescope_pointing` has already been updated. However, it uses the specific `Level1bModel`. This should be changed to using `jwst.datamodels.open` to allow other models.

### PR DESCRIPTION
**Pull request by emolter**
Resolves [JP-2090](https://jira.stsci.edu/browse/JP-2090)

Closes #6053

This PR addresses updating the set_velocity_aberration script to use DataModels instead of fits for opening and manipulating data.

**Checklist for maintainers**
- [x] added entry in `CHANGES.rst` within the relevant release section
- [x] updated or added relevant tests
- [ ] updated relevant documentation
- [x] added relevant milestone
- [x] added relevant label(s)
- [x] ran regression tests, post a link to the Jenkins job below.
      [How to run regression tests on a PR](https://github.com/spacetelescope/jwst/wiki/Running-Regression-Tests-Against-PR-Branches)
- [ ] Make sure the JIRA ticket is [resolved properly](https://github.com/spacetelescope/jwst/wiki/How-to-resolve-JIRA-issues)

### CODE REVIEW

**Comment by hbushouse** [comment]
Did you do a regression test run?  I'm not sure we have any tests that cover this script, but it can't hurt.

**Comment by emolter** [comment]
> Did you do a regression test run? I'm not sure we have any tests that cover this script, but it can't hurt.

I did not - the test coverage is inside lib/tests/test_velocity_aberration.py.  I will do a regression test run anyway, though

**Comment by emolter** [comment]
started [this Jenkins run](https://plwishmaster.stsci.edu:8081/job/RT/job/JWST-Developers-Pull-Requests/1265/)

**Review by hbushouse — COMMENTED** [review]
These code updates look fine. But before we approve and merge, I'm doing some research into JP-896, which suggests we might need/want to add another fix to this script, in order to have it use velocity information in the proper reference frame. That ticket suggests that these calculations should be using velocities that are in the heliocentric frame, but the keyword values we're using are actually in the Earth-centered frame. I'm trying to track down exactly what's going on with all of that.

**Comment by hbushouse** [comment]
Regression test run results are clean and JP-896 has been verified to no longer be an issue (fixed long ago in DMS build 7.8), so no further updates are needed to fix any old issues in this PR.

File: CHANGES.rst [hunk 1]
```diff
@@ -93,6 +93,12 @@ jump
 
 - Removed a unit test in Jump that was moved to STCAL to decrease
   the coupling of the two repos. [#8319]
+  
+ lib
```
Comment (hbushouse) [inline CHANGES.rst:97]:
```suggestion
lib
```

**Review by hbushouse — APPROVED** [review]

**Pull request merged**